### PR TITLE
Fixed problems in installing mistral

### PR DESCRIPTION
**Pull request by johandahlberg**
I added a prefix to the mistral version, and I also added a `touch /etc/mistral/database_setup.lock` to make sure that the database is not populated twice.

### CODE REVIEW

**Comment by johandahlberg** [comment]
I missed #30, so the first commit isn't really anything new. However I don't know what you think of 784061b as a solution to the [duplicate db problem](https://github.com/StackStorm/st2/issues/2062#issuecomment-146847237).

**Comment by jfryman** [comment]
Interesting error in CI though...

> stderr: touch: cannot touch `/etc/mistral/database_setup.lock': Permission denied

File: roles/mistral/tasks/sync.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - name: Sync database
-  shell: /opt/openstack/mistral/.venv/bin/python ./tools/sync_db.py --config-file /etc/mistral/mistral.conf
+  shell: /opt/openstack/mistral/.venv/bin/python ./tools/sync_db.py --config-file /etc/mistral/mistral.conf && touch /etc/mistral/database_setup.lock
```
Comment (armab) [inline roles/mistral/tasks/sync.yml]: Yeah, that's what I missed here https://github.com/armab/ansible-st2/commit/ce1177a17abafd0e681c57a4e7333c0e7c2f5455, good catch.

We can add `sudo: yes` for this task or go with `/opt/openstack/mistral/database_setup.lock` here and there.

**Comment by armab** [comment]
Thanks!

**Pull request merged**